src/app/clusters/level-control/level-control.cpp

```diff
@@ -657,6 +657,10 @@ bool emberAfLevelControlClusterMoveCallback(app::CommandHandler * commandObj, co
     auto & optionsMask     = commandData.optionsMask;
     auto & optionsOverride = commandData.optionsOverride;
 
+    if (rate == 0) {
+      rate.SetNull();
+    }
```

Comment:
The null value will cause the DefaultMoveRate to be used, which is writable and could also be zero. 

Recommend instead to set the rate to 1 instead of zero.

This is because:

>1.6.7.2.2. Rate Field
>
> This field SHALL indicate the rate of movement in units per second. The actual rate of movement **SHOULD be as close to this rate as the device is able**. If the Rate field is equal to null, then the value in DefaultMoveRate attribute SHALL be used. However, if the Rate field is equal to null and the DefaultMoveRate attribute is not supported, or if the Rate field is equal to null and the value of the DefaultMoveRate attribute is equal to null, then the device SHOULD move as fast as it is able. If the device is not able to move at a variable rate, this field MAY be disregarded.